hummingbird/ml/_utils.py

```diff
@@ -113,6 +113,18 @@ def xgboost_installed():
     return True
 
 
+def tvm_installed():
+    """
+    Checks that *TVM* is available.
+    """
+    try:
+        import tvm
+        from tvm import relay
+    except ImportError:
+        return False
```

Comment:
please add a test for this in `tests/test_no_extra_install.py`

Reply:
Ah, I forgot about this! Thanks! Added.